tox.ini

```
@@ -115,6 +117,7 @@ deps =
     boto3
     juju==3.5.2.1
     git+https://github.com/canonical/saml-test-idp.git
+    jubilant==0.4.0
```

Nit: sorted in alphabetical order :)
```suggestion
    git+https://github.com/canonical/saml-test-idp.git
    jubilant==0.4.0
    juju==3.5.2.1
```